tests/test_indexer.py

```diff
@@ -65,74 +64,43 @@
 #   5. Verify the structure and content of the index document
 #
 
+class HTTPInfo:
+    address = "127.0.0.1"
+    port = 8729
```

Comment:
can you file an improvement to search for an open port?  the ES server startup logic has an example.

Reply:
created issue #423